### PR TITLE
Update README & Fix Merge Error

### PR DESCRIPTION
**Pull request by kelshadarby**
### Type of change made:
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling (Styling to new or existing files)
- [ ] Testing
- [X] README

### Detailed Description
- Created a complete README
- Added get_all method to plant model that was accidentally removed during a merge conflict
### Why is this change required? What problem does it solve?
- To update README and add `get_all` so that the "view all plants" endpoint works
### Was any previously written code changed?
No previously written code was changed

### CODE REVIEW

File: app/models.py [hunk 1]
```diff
@@ -13,6 +13,9 @@ class Plant(db.Model):
     root_depth = db.Column(db.Integer)
     annual = db.Column(db.String(50))
 
+    def get_all():
```
Comment (mikez321) [inline app/models.py:16]: Thanks for adding this back in here!  Not sure where it got lost, probably at a merge conflict or something.  Maybe that means we should have a little more robust testing in the future??  (But we're aware of testing for this project)

File: README.md [hunk 2]
```diff
@@ -1,40 +1,224 @@
...
+__Get plant by ID (Plant Info)__
+This endpoint requires an ID to be passed in and it will return information about that plant
 ```
-oh-crop-be.herokuapp.com/api/v1/plants/2
+Request:
+https://oh-crop-be.herokuapp.com/api/v1/plants/2
+
+Response:
+{
+    "annual?": "Perennial",
```
Comment (mikez321) [inline README.md:107]: Oh, check this out!  That got changed in the DB, but never updated in the response.  Can you change this to something like annual/perennial or a/p or something like that??

File: README.md [hunk 3]
```diff
@@ -104,7 +104,7 @@ https://oh-crop-be.herokuapp.com/api/v1/plants/2
 
 Response:
 {
-    "annual": "Perennial",
+    "lifecycle": "Perennial",
```
Comment (mikez321) [inline README.md:107]: Nice, thanks for the update!

File: app/api/routes.py [hunk 4]
```diff
@@ -22,7 +22,7 @@ def get_plant(id):
                 'days_between_water': plant.water_frequency,
                 'days_to_harvest_from_seed': plant.harvest_time,
                 'root_depth_in': plant.root_depth,
-                'annual': plant.annual
+                'lifecycle': plant.annual
```
Comment (mikez321) [inline app/api/routes.py:25]: Looks great!

**Comment by mikez321** [comment]
Thanks for all that work on the Readme!  Looks great, we'll just need to remember to add more endpoints as we create them.  Also, thanks for changing annual? to lifecycle... makes more sense with our new DB layout!

File: README.md (hunk 2 above)
Comment (kelshadarby) [reply README.md]: Nice catch! I'll fix that :)

File: app/api/routes.py (hunk 4 above)
Comment (kelshadarby) [reply app/api/routes.py:25]: We should change this in the database as well

**Pull request merged**